### ISSUE TITLE
How to get logits of the network before softmax layer. 

### ISSUE THREAD

**Issue by qazi876** [YunYang1994/tensorflow-yolov3#548]
Hi Author. can you please tell me how to get logits in this model. and also explain the loss function, please.
